pandas/tests/computation/test_eval.py

```diff
@@ -339,8 +339,8 @@ def check_pow(self, lhs, arith1, rhs):
 
         if (is_scalar(lhs) and is_scalar(rhs) and
                 _is_py3_complex_incompat(result, expected)):
-            pytest.raises(AssertionError, tm.assert_numpy_array_equal,
-                          result, expected)
+            with pytest.raises(AssertionError):
+                tm.assert_numpy_array_equal(result, expected)
```

Comment:
didn't convert this on first pass since test is skipped on windows.  looking at it again, the assertion error check is used in lieu of a tm.assert_numpy_array_not_equal function so message check not appropriate.